response_operations_ui/common/surveys.py

```diff
@@ -0,0 +1,13 @@
+from enum import Enum
+
+
+class Surveys(Enum):
+    """Includes all surveys for the api"""
+    ASHE = "ASHE"
+    BLOCKS = "Blocks"
+    BRICKS = "Bricks"
+    FDI = "FDI"
+    NBS = "NBS"
+    PCS = "PCS"
+    QBS = "QBS"
+    SANDANDGRAVEL = "Sand&Gravel"
```

Comment:
Does this match the survey short name in the survey service?

I think this needs to be sorted because in the UI it shouldn't display without spaces.